web/olga/analytics/models.py

```diff
@@ -76,6 +76,20 @@ class InstallationStatistics(models.Model):
     )
 
     @classmethod
+    def get_stats_for_this_day(cls, edx_installation_object=None):
+        """
+        Provide model, for given installation object, that was created today, or None if it not exist.
```

Comment:
I would change doc string summary for something like that: 
```
      Provide statistic model instance for the given Edx installation.

      :param edx_installation_object: specific installation object.
      :return: statistic model instance if it is created today otherwise None
```